config/allconfig/alldecoders.go

```diff
@@ -419,6 +419,8 @@ var allDecoderSetups = map[string]decodeWeight{
 				p.c.UglyURLs = vv
 			case string:
 				p.c.UglyURLs = vv == "true"
+			case maps.Params:
+				p.c.UglyURLs = cast.ToStringMapBool(maps.CleanConfigStringMap(vv))
```

Comment:
We need a failing test case for this.

Also, I suspect you could handle this case in the `default`, but start with a test.

Reply:
@bep I've added test cases